adal/src/androidTest/java/com/microsoft/aad/adal/BrokerProxyTests.java

```diff
@@ -825,6 +844,10 @@ private Context getMockContext(final Signature signature, final String brokerPac
         PackageManager mockPackageManager = getPackageManager(signature, brokerPackageName, permissionStatus);
         when(mockContext.getPackageManager()).thenReturn(mockPackageManager);
         when(mockContext.getPackageName()).thenReturn(contextPackageName);
+        List<ResolveInfo> mockList = new ArrayList<>();
+        mockList.add(new ResolveInfo());
+        int size = mockList.size();
```

Comment:
Here in Line 848 below, mockList is used in mocking the returns of PackageManager.queryIntentActivities().